exporters/otlp/include/opentelemetry/exporters/otlp/otlp_grpc_metric_exporter_options.h

```diff
@@ -19,8 +19,13 @@ namespace otlp
  */
 struct OtlpGrpcMetricExporterOptions : public OtlpGrpcExporterOptions
 {
-  opentelemetry::sdk::metrics::AggregationTemporality aggregation_temporality =
-      opentelemetry::sdk::metrics::AggregationTemporality::kDelta;
+
+  // Preferred Aggregation Temporality
+  sdk::metrics::AggregationTemporality aggregation_temporality =
+      sdk::metrics::AggregationTemporality::kCumulative;
+
+  // opentelemetry::sdk::metrics::AggregationTemporalitySelector aggregation_temporality_selector =
```

Comment:
What is this for?

Reply:
Leftover from debugging, shouldn't be there. Removed thanks :)